src/bflib_video.c

```diff
@@ -437,15 +429,15 @@ TbResult LbScreenUpdateIcon(void)
     SDL_SysWMinfo wmInfo;
```

Comment:
Not used anymore?

Reply:
I guess not, icon is taken by winduz from exe resources, so setting the icon is unnecessary.

Reply:
ok